TeamCode/src/main/java/org/firstinspires/ftc/teamcode/DriveABot.java

```diff
@@ -19,11 +19,11 @@ public class DriveABot extends LinearOpMode
     DcMotor motorRight = null;
 
     // Drive mode constants
-    public static final int TANK_DRIVE = 0;
-    public static final int ARCADE_DRIVE = 1;
+    public static final int TANK_DRIVE = 1;
+    public static final int ARCADE_DRIVE = 0;
     public static final int SPLIT_ARCADE_DRIVE = 2;
     public static final int GAME_DRIVE = 3;
-    public int driveMode = ARCADE_DRIVE;
+    public int driveMode = TANK_DRIVE;
```

Comment:
Why are these changing?

Reply:
We wanted to change the drive mode for the drive-a-bots. I don't think we are using that code anymore but either way it was what got pushed to the DAB.